application/server/main_test.go

```diff
@@ -427,40 +425,6 @@ func TestGetItems_200(t *testing.T) {
 	assert.Equal(t, uint(5), (*results)[0].ClassReq)
 }
 
-func TestGetItems_500(t *testing.T) {
```

Comment:
Note that these tests are no longer needed because the only time the endpoint will fail is if there are no items (which is not easily testable)